plugins/woocommerce/src/Internal/Orders/POSOrderUtil.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * POSOrderUtil class file.
+ */
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Internal\Orders;
+
+use WC_Abstract_Order;
+
+/**
+ * Helper class for POS order related functionality.
+ *
+ * @internal Just for internal use.
+ */
+class POSOrderUtil {
```

Comment:
We could expand the abbreviation here, and some folks might not be exposed to the POS domain.
For email templates, keeping abbreviations is OK, I think - it's long names already.

Reply:
Yea makes sense, renamed classes/files/usage in https://github.com/woocommerce/woocommerce/pull/57573/commits/a44124ef7f55aed7e4ae8902b18dab4d27b4b400.